nyaa/api_torznab.py

```diff
@@ -101,71 +112,113 @@ def torznab_caps():
     return render_torznab_caps()
 
 
-def torznab_search():
-    page_size = app.config['RESULTS_PER_PAGE']
+def get_categories_filter(cat):
+    categories = []
+    if cat:
+        for item in cat.split(','):
+            # Parse 1_1 format for compatibility
```

Comment:
How will that work? Considering that an empty `cat` has special meaning. empty list vs list of unknown categories.
Which is why it's checked whether categories is empty inside the `if cat:`, coz if it's empty then the user did specify categories but they didn't translate to actual categories and an empty result set should be returned. If the user didn't specify a category, then it should search all categories.